packages/create-platformatic/src/db/create-db-cli.mjs

```diff
@@ -166,7 +166,7 @@ const createPlatformaticDB = async (_args) => {
   if (!runPackageManagerInstall) {
     logger.warn(`You must run the following commands in the project folder to complete the setup:
     - ${pkgManager} install
-    - npx platformatic db schema config > ./platformatic.config.schema.json
+    - npx platformatic db schema config > ./platformatic.db.schema.json
```

Comment:
@marcopiraccini As I [mentioned here](https://github.com/platformatic/platformatic/pull/488#discussion_r1049393012), I don't think this is correct. I've just tested running:

```bash
npx platformatic db schema config
```

And it generates the `platformatic.db.schema.json` file — it doesn't send anything to stdout. That means the step suggested here should be:

```
- npx platformatic db schema config
```